media-video/mpv/files/mpv-0.14.0-support-GNU-__thread.patch

```diff
@@ -0,0 +1,33 @@
+See https://github.com/mpv-player/mpv/issues/2631
```

Comment:
Was this merged? The issue has quite a long discussion, so if possible we'd rather see a commit link here.